commit 17bf54576077885e4c39948555879c2a42ec945e (refs/changes/56/256/3)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2016-08-09T10:14:09-07:00 (3 years, 2 months ago)
    
    TOOLS-1515 Add step to update boilerplate intro when importing an existing GH repo (line1 missing char)

diff --git a/docs/user/README.md b/docs/user/README.md
index 66c0c27..f04ca62 100644
--- a/docs/user/README.md
+++ b/docs/user/README.md
@@ -262,10 +262,11 @@ thing is to have one of them do this for you.
 Once a repository has been imported:
 
 * **All subsequent pushes should go through Gerrit.**  People should stop
-  pushing to the GitHub copy.  Please notify people who might push to the
-  repository (e.g., by emailing "devs").  Ideally, we would turn off the ability
-  to push to GitHub, but we have not written any tooling or documentation for
-  this yet.
+  pushing to the GitHub copy.  Please notify the internal teams by emailing
+  "devs" and notify the external developer community by updating the repo
+  contribution guidelines ([boilerplate](https://mo.joyent.com/eng/blob/master/README.md)).
+  Ideally, we would turn off the ability to push to GitHub, but we have not
+  written any tooling or documentation for this yet.
 * Any changes pushed to "master" in Gerrit will be replicated to GitHub.
 * GitHub remains the repository of record for automated builds and other
   tooling.
